src/main/java/org/broadinstitute/hellbender/engine/spark/FindAssemblyRegionsSpark.java

```diff
@@ -225,12 +231,16 @@ public ReadlessAssemblyRegion apply(@Nullable AssemblyRegion input) {
                         });
     }
 
-    private static AssemblyRegion toAssemblyRegion(Shard<GATKRead> shard, SAMFileHeader header) {
```

Comment:
Has this affected any of the test outputs? Could we try building a test where we reduce the number of reads per start position to 1 just to demonstrate this is behaving consistently?